deployments/liqo/values.yaml

```diff
@@ -305,10 +305,14 @@ fabric:
   image:
     # -- Image repository for the fabric pod.
     name: "ghcr.io/liqotech/fabric"
-    # -- Custom version for the fabric image. If not specified, the global tag is used.
+    # -- Custom version for the fabric image. If not specified, the global tag |is used.
     version: ""
   # -- Extra tolerations for the fabric daemonset.
   tolerations: []
+  config:
+    # -- Enabe/Disable the full masquerade mode for the fabric pod.
+    # It means that all traffic will be masquerade using the first external cidr IP, instead of using the pod IP.
```

Comment:
I would add a short description indicating the pros and cons of this flag and when it can be helpful to (1-2 lines)